docs/config.json

```
@@ -578,6 +578,14 @@
 	  "title": "Using Dynamic Resources",
 	  "slug": "/management/dynamic-resources/",
 	  "entries": [
+        {
+          "title": "Managing users and roles",
```

```suggestion
          "title": "Managing Users and Roles",
```
We use title case in page titles